plugin-dev/Scripts/upload-debug-symbols-win.bat

```diff
@@ -33,11 +33,10 @@ if "%TargetPlatform%"=="Win64" (
 )
 
 call :ParseIniFile "%ConfigPath%\DefaultEngine.ini" /Script/Sentry.SentrySettings UploadSymbolsAutomatically UploadSymbols
-echo Sentry: Automatic symbols upload is '%UploadSymbols%' from plugin settings
```

Comment:
Won't we miss that log?

Reply:
If symbol upload flag is unspecified/disabled in settings or overridden by env variable we'll print a corresponding message later on anyway, so keeping this message here doesn't seem to be necessary any longer.